main.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <stdio.h>
+
+
+int main(int argc, char *argv[]) {
+    // std::ofsteam configfile;
+   // configfile.open("known_recipients.txt");
+
+    //std::string argument1 = "";
+
+//std::string user_input = "";
+
+//Location
+
+//if(argument = init)
+    std::string first_and_last ="", email = "", cypher = "", timezone, path ="";
+
+    std::cout << "Enter your first and last name, please" << std::endl;
+    std::getline(std::cin, first_and_last);
+
+        while(first_and_last == ""){
+        std::cout << "Please enter your first and last name to continue" << std::endl;
```

Comment:
your code makes sense and I think is good for the init portion. No the problem you have to consider is. How can I edit only one field in the file. While that sounds like a daunting task possibly try not to get stuck in the idea that you have to just change the existing file. At the end of the process you just need a file with the correct information in it.